File: official/docs/java/current/billing/retrieve-stripe-public-key.java
package billing;

import com.easypost.exception.EasyPostException;
import com.easypost.service.EasyPostClient;

public class RetrieveStripePublicKey {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient("EASYPOST_API_KEY");

        Object response = client.betaReferralCustomer.retrieveEasypostStripeApiKey();

        System.out.println(response);
    }
}

File: official/docs/golang/current/billing/retrieve-stripe-public-key.go
package example

import (
	"fmt"

	"github.com/EasyPost/easypost-go/v5"
)

func retrieveStripePublicKey() {
	client := easypost.New("EASYPOST_API_KEY")

	response, _ := client.RetrieveEasypostStripeApiKey()

	fmt.Println(response)
}
